gnucash/gnome/dialog-assoc.c

```diff
@@ -537,7 +537,9 @@ row_selected_bus_cb (GtkTreeView *view, GtkTreePath *path,
             return;
         }
 
-        ret_uri = gnc_assoc_get_uri_dialog (GTK_WINDOW(assoc_dialog->window), _("Change a Business Association"), uri);
+/* Translators: This is the title of a dialog box for associating an external
+ * file or URI with the current bill, invoice, transaction, or voucher. */
```

Comment:
Results in
```
#. Translators: This is the label of a dialog box that lists all of the
#. * transaction that have files or URIs associated with them.

```
Better no beginning comment continuation star in translator comments. Same issue below, too